include/rdma/fi_eq.h

```diff
@@ -74,8 +72,8 @@ struct fid_wait {
 };
 
 struct fi_mutex_cond {
-	pthread_mutex_t		*mutex;
-	pthread_cond_t		*cond;
+	struct pthread_mutex_t	*mutex;
+	struct pthread_cond_t	*cond;
```

Comment:
If the reason is osd.h not being part of external headers, we may need to consider either move them to the external headers or remove os dependent references from rdma/*.